db/migrations/81_rename_groups_attempts_to_attempts.sql

```diff
@@ -73,10 +73,8 @@ CREATE TRIGGER `before_update_items_items` BEFORE UPDATE ON `items_items` FOR EA
         UPDATE `attempts` SET `result_propagation_state` = 'to_be_recomputed'
         WHERE `item_id` = OLD.`parent_item_id`;
 
-        IF (OLD.`parent_item_id` != NEW.`parent_item_id`) THEN
-            UPDATE `attempts` SET `result_propagation_state` = 'to_be_recomputed'
-            WHERE `item_id` = NEW.`parent_item_id`;
-        END IF;
+        UPDATE `attempts` SET `result_propagation_state` = 'changed'
+        WHERE `item_id` = OLD.`child_item_id` OR `item_id` = NEW.`child_item_id`;
```

Comment:
If you do so, the previous request on the parent may only be done if `(OLD.`parent_item_id` != NEW.`parent_item_id`)`.